Form/Handler/ListFormHandler.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace LAG\AdminBundle\Form\Handler;
+
+use BlueBear\BaseBundle\Entity\Behaviors\Id;
+use LAG\AdminBundle\Admin\Behaviors\EntityLabel;
+use Symfony\Component\Form\FormInterface;
+
+class ListFormHandler
```

Comment:
Is this linked to the batch form type? Is the naming deliberately less specific?